src/pages/clubs/redux/index.tsx

```diff
@@ -59,12 +59,23 @@ const reduxClub: React.FC<any> = ({data}) => {
 
 export default reduxClub
 
+export const reduxClubQuery = groq`*[_type == 'resource' && slug.current == "build-business-oriented-portfolio"][0]{
```

Comment:
this was being imported but I didn't understand why since it was only used in one spot that I could tell.